javaparser-core/src/main/java/com/github/javaparser/ast/visitor/CloneVisitor.java

```diff
@@ -429,6 +429,15 @@ public Node visit(WildcardType _n, Object _arg) {
 		return r;
 	}
 
+    @Override
+    public Node visit(UnknownType _n, Object _arg) {
+        Comment comment = cloneNodes(_n.getComment(), _arg);
```

Comment:
If so it should dumped in the DumpVisitor, if not is should be prevent to have a comment associated